src/gui/statusbar.cpp

```diff
@@ -224,6 +234,14 @@ void StatusBar::refreshStatusBar()
     updateConnectionStatus(sessionStatus);
     updateDHTNodesNumber(sessionStatus);
     updateSpeedLabels(sessionStatus);
+    updateFreeSpaceOnDisk();
+}
```

Comment:
@KingLucius:  Is this bracket supposed to be here?  

It does not look like there was one here before.